### ISSUE TITLE
generic_carver/binwalk problems

### ISSUE THREAD

**Issue by jstucke** [fkie-cad/fact_extractor#90]
In the latest version, binwalk seems to need an additional parameter `--run-as=root` in order to unpack files.

```
Extractor Exception: Binwalk extraction uses many third party utilities, which may not be secure. If you wish to have extraction utilities executed as the current user, use '--run-as=root' (binwalk itself must be run as root).
----------------------------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/binwalk-2.3.3+fa0c0bd-py3.6.egg/binwalk/core/module.py", line 258, in __init__
    self.load()
  File "/usr/local/lib/python3.6/dist-packages/binwalk-2.3.3+fa0c0bd-py3.6.egg/binwalk/modules/extractor.py", line 147, in load
    raise ModuleException("Binwalk extraction uses many third party utilities, which may not be secure. If you wish to have extraction utilities executed as the current user, use '--run-as=%s' (binwalk itself must be run as root)." % user_info.pw_name)
binwalk.core.exceptions.ModuleException: Binwalk extraction uses many third party utilities, which may not be secure. If you wish to have extraction utilities executed as the current user, use '--run-as=root' (binwalk itself must be run as root).
----------------------------------------------------------------------------------------------------
```

**Comment by wideglide** [comment]
Might be worth pinning binwalk to 2.3.2 for a bit until they get this newest patch sorted out.
- for the moment, it seems the internal tests are broken
ReFirmLabs/binwalk/issues/566
ReFirmLabs/binwalk/issues/569

**Comment by jstucke** [comment]
Resolved by #91
